bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/TprimeBToTH_Hbb_LH/TprimeBToTH_Hbb_LH_MT600_MH75/TprimeBToTH_Hbb_LH_MT600_MH75_run_card.dat

```diff
@@ -0,0 +1,267 @@
+#*********************************************************************
+#                       MadGraph5_aMC@NLO                            *
+#                                                                    *
+#                     run_card.dat MadEvent                          *
+#                                                                    *
+#  This file is used to set the parameters of the run.               *
+#                                                                    *
+#  Some notation/conventions:                                        *
+#                                                                    *
+#   Lines starting with a '# ' are info or comments                  *
+#                                                                    *
+#   mind the format:   value    = variable     ! comment             *
+#                                                                    *
+#   To display more options, you can type the command:               *
+#      update full_run_card                                          *
+#*********************************************************************
+#
+#*********************************************************************
+# Tag name for the run (one word)                                    *
+#*********************************************************************
+tag_1     = run_tag ! name of the run
+#*********************************************************************
+# Number of events and rnd seed                                      *
+# Warning: Do not generate more than 1M events in a single run       *
+#*********************************************************************
+100000 = nevents ! Number of unweighted events requested
+0   = iseed   ! rnd seed (0=assigned automatically=default))
+#*********************************************************************
+# Collider type and energy                                           *
+# lpp: 0=No PDF, 1=proton, -1=antiproton, 2=photon from proton,      *
+#                                         3=photon from electron     *
+#*********************************************************************
+1        = lpp1    ! beam 1 type
+1        = lpp2    ! beam 2 type
+6500.0     = ebeam1  ! beam 1 total energy in GeV
+6500.0     = ebeam2  ! beam 2 total energy in GeV
+# To see polarised beam options: type "update beam_pol"
+
+#*********************************************************************
+# PDF CHOICE: this automatically fixes also alpha_s and its evol.    *
+#*********************************************************************
+'lhapdf' = pdlabel     ! PDF set
+320900 = lhaid
```

Comment:
why not putting default_pdf as mentioned on the twiki ? i.e. why not using 325500 ?

Reply:
As far as I knew, 303600 had been used with CP5 in 2017 and 18, so I assumed switching to the 4f-version of that PDF would be the preferred option. Would putting $DEFAULT_PDF_SETS correctly choose 263400 for 2016 production (where the 4f-version of 263000 for TuneCUEP8M1 should be used)?

Reply:
I have changed the default PDF in the run_card and the script to generate the other mass points to 325500. For 2016,  263000 is still used at the moment.